### PR TITLE
updating helpers docs

### PR DESCRIPTION
**Pull request by iruzevic**
- documentation for all scripts and removing the script exports that are not used and should not be used by the users.
- adding usage and output examples so we can have documentation in one place.
- fixed the import order so it correlates to the folder structure for easier maintenance.
- all copied from the docs so no need for details check.

In combination with this PR on docs - https://github.com/infinum/eightshift-docs/pull/110

### CODE REVIEW

File: scripts/editor/get-actions.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ const mediaPropsAction = (setAttributes, key) => {
 
 /**
  * Create attributes actions from blocks manifest.json.
+ * This helper is deprecated and should not be used anymore.
```
Comment (dingo-d) [inline scripts/editor/get-actions.js:131]: Maybe we can use https://jsdoc.app/tags-deprecated.html for this?

File: scripts/plugins/yoast-seo.js [hunk 2]
```diff
@@ -115,6 +115,17 @@ class YoastSEOCustomData {
 	}
 }
 
+/**
+ * This helper will search all blocks and components manifests and find attributes that have "seo": "true" key.
```
Comment (dingo-d) [inline scripts/plugins/yoast-seo.js:119]:
```suggestion
 * This helper will search all blocks and component manifests and find attributes that have "seo": "true" key.
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left some minor comments

File: scripts/editor/get-actions.js [hunk 3]
```diff
@@ -137,6 +138,8 @@ const mediaPropsAction = (setAttributes, key) => {
  * @param {object} props    - Block props so we can get `setAttributes` method.
  * @param {object} manifest - Block manifest.json so we can get all attributes.
  *
+ * @deprecated We stopped using getActions helper and covered to the native way of writing setAttributes
```
Comment (goranalkovic-infinum) [inline scripts/editor/get-actions.js:141]:
```suggestion
 * @deprecated This helper is not used anymore. Write regular `setAttributes` instead.
```

File: scripts/editor/get-actions.js (hunk 1 above)
Comment (goranalkovic-infinum) [inline scripts/editor/get-actions.js:131]:
```suggestion
 * _This helper is deprecated and should not be used anymore._
```

File: scripts/editor/get-options.js [hunk 4]
```diff
@@ -64,13 +84,24 @@ export const getOption = (key, attributes, manifest, isColor = false) => {
 }
 
 /**
- * Use this hook to filter the global colors out of the component or block manifest
+ * Use this hook to filter the global colors out of the component or block manifest.
+ * This function is deprecated and getOption should be used.
```
Comment (goranalkovic-infinum) [inline scripts/editor/get-options.js:88]:
```suggestion
 * _This function is deprecated._
 *
 * @deprecated Use `getOption` instead.
```

File: scripts/editor/output-css-variables.js [hunk 5]
```diff
@@ -480,9 +567,19 @@ export const prepareVariableData = (globalBreakpoints) => {
 }
 
 /**
- * Returns a unique ID.
+ * Returns a unique ID generally used for css variables.
```
Comment (goranalkovic-infinum) [inline scripts/editor/output-css-variables.js:570]:
```suggestion
 * Returns a unique ID, generally used with CSS variable generation.
```

File: scripts/editor/override-inner-block-attributes.js [hunk 6]
```diff
@@ -1,3 +1,25 @@
+/**
+ * Used to set attributes on all innerBlocks. This value will be stored in the block editor store and set to a block.
```
Comment (goranalkovic-infinum) [inline scripts/editor/override-inner-block-attributes.js:2]:
```suggestion
 * Sets attributes on all `innerBlocks`. This value will be stored in the Block editor store and set to a block.
```

File: scripts/editor/override-inner-block-attributes.js [hunk 7]
```diff
@@ -18,6 +40,24 @@ export const overrideInnerBlockAttributes = (select, clientId, attributesObject
 	});
 };
 
+/**
+ * Used to set attributes on all innerBlocks preset only for simple wrapper setup. This value will be stored in the block editor store and set to a block.
```
Comment (goranalkovic-infinum) [inline scripts/editor/override-inner-block-attributes.js:44]:
```suggestion
 * Sets attributes on all `innerBlocks`, with Simple wrapper options preset. This value will be stored in the Block editor store and set to a block.
```

File: scripts/helpers/escape-string.js [hunk 8]
```diff
@@ -1,10 +1,20 @@
...
  *
  * @return Escaped string.
+ *
+ * Usage:
+ * ```js
+ * escapeString.escapeString('Speci^al stri&n$g');
+ * ```
+ *
+ * Output:
+ * ```js
```
Comment (goranalkovic-infinum) [inline scripts/helpers/escape-string.js:16]:
```suggestion
 * ```
```

File: scripts/plugins/yoast-seo.js [hunk 9]
```diff
@@ -115,6 +115,17 @@ class YoastSEOCustomData {
 	}
 }
 
+/**
+ * This helper will search all blocks and component manifests and find attributes that have "seo": "true" key.
```
Comment (goranalkovic-infinum) [inline scripts/plugins/yoast-seo.js:119]:
```suggestion
 * Searches all blocks and component manifests and returns attributes that have `"seo": "true"` set.
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
Left some suggestions

**Review by dingo-d — DISMISSED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Review by volfkarlo — DISMISSED** [review]

File: scripts/helpers/cookies.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
 /**
- * Cookies helper
+ * Helper to set and unset cookies.
```
Comment (mbmjertan) [inline scripts/helpers/cookies.js:2]: Perhaps listing the available helper methods for time (`setOneDay` etc.) would be useful.

File: scripts/helpers/text-helpers.js [hunk 11]
```diff
@@ -1,15 +1,22 @@
...
  *
- * Example: _I'm a tr...d string_.
+ * @param {string} input             - String to slice.
+ * @param {number} maxLength         - Number to characters to slice.
+ * @param {string} [separator='...'] - Separator to insert.
  *
- * @param {string} input             - Input string.
- * @param {Number} maxLength         - Maximum allowed length of the string.
- * @param {string} [separator='...'] - Separator to use.
+ * @returns {string}
```
Comment (mbmjertan) [inline scripts/helpers/text-helpers.js:8]:
```suggestion
 * @returns {string} Truncated string.
```

**Review by mbmjertan — CHANGES_REQUESTED** [review]

File: scripts/helpers/text-helpers.js [hunk 12]
```diff
@@ -1,15 +1,22 @@
 /**
- * Truncates a string to a certain length, separating it in the middle.
+ * Slices the string and inputs the provided separator after the specified characters.
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply scripts/helpers/text-helpers.js]: Please create a issue for this @mbmjertan

**Pull request merged**